tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=product-description]', { timeout: 20000 })
```

Comment:
Since "product description" is actually a field of a product, I would use a different name to refer to the product data (i.e. name, description, attributes, price, etc.). Maybe "product-data", or "product".